crates/cargo-remove/src/bin/cargo/commands/remove.rs

```diff
@@ -1,8 +1,7 @@
 use cargo::util::command_prelude::*;
-use cargo_remove::ops::cargo_remove::remove;
-use cargo_remove::ops::cargo_remove::DepKind;
-use cargo_remove::ops::cargo_remove::DepTable;
-use cargo_remove::ops::cargo_remove::RemoveOptions;
```

Comment:
I actually really dislike use of `{}` with imports.  It makes it more annoying to evolve the code.

Reply:
Okay, updated to use single item imports.